test-lib-openshift.sh

```diff
@@ -788,15 +794,24 @@ function ct_os_test_template_app_func() {
 
   local ip
   local check_command_exp
+  local image_id
+
+  # get image ID from the deployment config
+  image_id=$(oc get "deploymentconfig.apps.openshift.io/${service_name}" -o custom-columns=IMAGE:.spec.template.spec.containers[*].image | tail -n 1)
 
   ip=$(ct_os_get_service_ip "${service_name}")
   # shellcheck disable=SC2001
-  check_command_exp=$(echo "$check_command" | sed -e "s/<IP>/$ip/g")
+  check_command_exp=$(echo "$check_command" | sed -e "s/<IP>/$ip/g" -e "s|<SAME_IMAGE>|${image_id}|g")
 
   echo "  Checking APP using $check_command_exp ..."
   local result=0
   eval "$check_command_exp" || result=1
 
+  echo "  Information about the image we work with:"
+  oc get deploymentconfig.apps.openshift.io/"${service_name}" -o yaml | grep lastTriggeredImage
+  # for s2i builds, the resulting image is actually in the current namespace
+  oc get isimage -n "${namespace}" "${image_id##*/}" -o yaml || oc get isimage "${image_id##*/}" -o yaml
+
```

Comment:
This seems like it would be nice to have as a separate function

Reply:
good idea, will change it

Reply:
Done.